src/pages/Skills/SkillsList/SimpleAccordion/index.js

```diff
@@ -135,7 +135,7 @@ export default function SimpleAccordion({ skills }) {
                     <List component="div" disablePadding>
                         {
                           skill.examples.map((example) => {
-                            return <div><ul>{example}</ul></div>
+                            return <div key={example}><ul>{example}</ul></div>;
```

Comment:
had to add this to pass the husky check, but imagine this will change quite a bit on LFH-103 anyway

Reply:
nice I also changed it because of the husky check here https://github.com/Caitlin-cooling/loofah-frontend/pull/50/files so we might have conflicts

Reply:
ok cool let's keep an eye on this when we merge 👍